linkedevents/settings.py

```diff
@@ -259,6 +261,12 @@ def get_git_revision_hash() -> str:
 ACCOUNT_LOGOUT_ON_GET = True
 SOCIALACCOUNT_ADAPTER = "helusers.adapter.SocialAccountAdapter"
 
+# Default organization to which a user will be assigned if user doesn't
+# have any organization.
+USER_DEFAULT_ORGANIZATION_ID = env("USER_DEFAULT_ORGANIZATION_ID")
```

Comment:
What's the difference between `DEFAULT_ORGANIZATION_ID` and `USER_DEFAULT_ORGANIZATION_ID` atm? 🤔 Do we have a case where these need to be separate values?

Reply:
Ideally they should be the same value yes. `USER_DEFAULT_ORGANIZATION_ID` sort of controls whether the "put user without an organization into the default organization" feature is enabled. That's probably not optimal. Maybe I'll still refactor this so that we have a setting like `ENABLE_USER_DEFAULT_ORGANIZATION` and `USER_DEFAULT_ORGANIZATION_ID` (and the default org would automatically get created if it doesn't exist. :thinking: